src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/NotificationService.java

```diff
@@ -79,35 +73,35 @@ public AlertStatus getAlerts() {
             alertStatus.setBackupRestoreRunningSince(backupRestoreStartedAt);
             alertStatus.setCurrentTime(OffsetDateTime.now(ZoneOffset.UTC));
         }
-        if (isSecurityServerFullyInitialized()) {
+
+        try {
             alertStatus.setGlobalConfValid(isGlobalConfValid());
-            alertStatus.setSoftTokenPinEntered(isSoftTokenPinEntered());
-        } else {
+            alertStatus.setGlobalConfValidCheckSuccess(true);
+        } catch (CodedException e) {
```

Comment:
Could just catch any `Exception`, probably only `CodedExceptions` should occur now, but we might not have realized a possibility of some other exceptions like RuntimeExceptions or some future changes may make that possible - and we probably want those to also result in setGlobalConfValidCheckSuccess(false)?

Reply:
The implementation was updated so that `Exception` is catched. It's possible that `globalConfFacade.verifyValidity()` [throws other exceptions](https://github.com/nordic-institute/X-Road/blob/develop/src/common-verifier/src/main/java/ee/ria/xroad/common/conf/globalconf/GlobalConf.java#L64) too (e.g. `RuntimeException`). Please see the first comment regarding the updated logic.